src/libraries/System.Private.Xml/tests/XmlSchema/TestFiles/TestData/issue_60543_1.xsd

```diff
@@ -0,0 +1,12 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"> 
```

Comment:
please rename the files into something like:
`nmtokens_restriction_multiple_enum_values` (or find better descriptive names) and similar for other files

Reply:
also might be good idea to write the expected result in the file name